qualtran/bloqs/basic_gates/su2_rotation.py

```diff
@@ -106,28 +106,18 @@ def from_matrix(mat: NDArray[np.complex_]) -> 'SU2RotationGate':
 
         return SU2RotationGate(theta, phi, lambd, alpha)
 
-    def add_my_tensors(
-        self,
-        tn: 'qtn.TensorNetwork',
-        tag: Any,
-        *,
-        incoming: Dict[str, 'SoquetT'],
-        outgoing: Dict[str, 'SoquetT'],
-    ):
+    def my_tensors(
+        self, incoming: Dict[str, 'ConnectionT'], outgoing: Dict[str, 'ConnectionT']
+    ) -> List['qtn.Tensor']:
         import quimb.tensor as qtn
 
-        tn.add(
+        return [
             qtn.Tensor(
                 data=self.rotation_matrix,
-                inds=(outgoing['q'], incoming['q']),
-                tags=[self.pretty_name(), tag],
+                inds=[(outgoing['q'], 0), (incoming['q'], 0)],
+                tags=[str(self)],
             )
-        )
-
-    def _unitary_(self):
-        if self.is_symbolic():
-            return None
-        return self.rotation_matrix
+        ]
```

Comment:
Ditto: Looks related to cirq-interop changes in `Controlled` gate that can be pulled out in a separate PR.

Reply:
reverted